app/src/debug/AndroidManifest.xml

```diff
@@ -3,7 +3,11 @@
     xmlns:tools="http://schemas.android.com/tools"
     package="com.github.sdp.ratemyepfl">
 
-    <application tools:ignore="MissingApplicationIcon">
+    <uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE"/>
```

Comment:
DId you find a way to get rid of the check warning in this line?

Reply:
Done.